config/rbac/role.yaml

```diff
@@ -40,6 +40,15 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
- May I know why we need a handle on subscription resource?
- I assume it's due to minor version change, however just want to understand the background in detail if possible.

Reply:
For upgrading minor versions we need to provide a dependency and code changes in the deployer.
So this change `"4.10.9 || 4.11.5"` updates the deployer with the code changes necessary. odf doesn't get upgraded yet.
The deployer acts on the subscription to change the ODF channel to 4.11after which odf upgrades